editor/src/core/workers/parser-printer/parser-printer-transpiling.ts

```diff
@@ -110,15 +114,38 @@ function babelRewriteJSXArbitraryBlockCode(
       }
     }
   }
-  function handleByPosition(path: BabelTraverse.NodePath<any>): void {
+  function tagNameForNodeName(
+    nodeName:
+      | BabelTypes.JSXIdentifier
+      | BabelTypes.JSXMemberExpression
+      | BabelTypes.JSXNamespacedName,
+  ): string {
+    if (BabelTypes.isJSXIdentifier(nodeName)) {
+      return nodeName.name
+    } else if (BabelTypes.isJSXMemberExpression(nodeName)) {
+      return nodeName.property.name
+    } else {
+      return `${nodeName.namespace.name}.${nodeName.name}`
+    }
+  }
+  function handleByPositionOrName(path: BabelTraverse.NodePath<BabelTypes.JSXElement>): void {
     const pathLocation: {
       line: number
       column: number
-    } = path.node.loc.start
-    const foundElementWithin = elementsWithin.find((e) => {
+    } = path.node.loc?.start ?? { line: -1, column: -1 }
```

Comment:
Are these being set to `-1` to make the `find` lower down fail?

Reply:
Yeah. So `path.node.loc` is nullable, but we weren't capturing that because we were previously using `any`. I think in practice it won't be `null`, but either way this means the find by location part will fail.